src/styles/normalize.scss

```diff
@@ -0,0 +1,10 @@
+html,
+body,
+img,
+p,
+a,
+div {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
```

Comment:
again, font-family should be set for the entire page (e.g html or body tags) and probably inside another file (for example a separate file `fonts`)